### PR TITLE
docs: add `i18n` documentation to subclasses of `disnake.Client`

### LINKED ISSUE #652: docs: `i18n` property is not documented on `commands.Bot`
### Summary

`i18n` property missing on `commands.Bot` class

### Reproduction Steps

Clone the repo, build the docs, GOTO `docs/_build/html/ext/commands/api.html` and open it in your browser (Alternatively GOTO https://docs.disnake.dev/en/stable/ext/commands/api.html). You'll see that `i18n` property is missing on `commands.Bot` class reference, while physically exist in library.

### Minimal Reproducible Code

```python
N/A
```

### Expected Results

See the `i18n` property :P

### Actual Results

There is no `i18n` property 

### Intents

N/A

### System Information

```markdown
disnake v2.5.1 and sphinx 4.5.0, other info is not applicable
```

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by ItsAleph**
## Summary

Fixes #652

Due to [sphinx-doc/sphinx#741](https://github.com/sphinx-doc/sphinx/issues/741), Sphinx's autodoc did not documented attributes of inherited classes until Sphinx 3.5. Looks like this issue popped up yet again, thus ~~`disnake.Client.i18n` was turned into property~~ docstring from `disnake.Client` was copied into `commands.Bot` and `commands.InteractionBot`

## Checklist

- [x] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `task lint`
    - [x] I have type-checked the code by running `task pyright`
- [x] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: disnake/ext/commands/bot.py [hunk 1]
```diff
@@ -252,6 +258,11 @@ class InteractionBot(InteractionBotBase, disnake.Client):
         Only applicable if the ``localization_provider`` parameter is not provided.
         Defaults to ``False``.
 
+        .. versionadded:: 2.5
+    i18n: :class:`.LocalizationProtocol`
```
Comment (onerandomusername) [inline disnake/ext/commands/bot.py:262]: missing a new line to be consistent
```suggestion
        .. versionadded:: 2.5

    i18n: :class:`.LocalizationProtocol`
```

File: changelog/652.doc.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+:class:`.ext.commands.Bot` and :class:`.ext.commands.InteractionBot` class references now document :class:`Client.i18n` attribute
```
Comment (onerandomusername) [inline changelog/652.doc.rst:1]: Needs a period at the end.
```suggestion
|commands| Document the ``i18n`` attribute on :class:`.ext.commands.Bot` and :class:`.ext.commands.InteractionBot` classes.
```

**Review by onerandomusername — CHANGES_REQUESTED** [review]
Two comments, other than it looks good.

**Review by onerandomusername — APPROVED** [review]
lgtm

**Review by shiftinv — APPROVED** [review]

**Pull request merged**